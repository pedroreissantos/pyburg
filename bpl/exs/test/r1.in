qw
er
ty
ui
op

