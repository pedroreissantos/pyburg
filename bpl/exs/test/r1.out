qw
er
ty
er
ty
ui
op
ui
op

